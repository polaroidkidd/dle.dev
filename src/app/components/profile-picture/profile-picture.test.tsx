import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { ProfilePicture } from './index';


test('render ProfilePicture with text', () => {
  const tree = renderer.create(<ProfilePicture imageSrc="empty"/>).toJSON();
  expect(tree).toMatchSnapshot();
});
