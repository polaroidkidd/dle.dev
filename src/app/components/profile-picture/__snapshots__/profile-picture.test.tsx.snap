// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render large ProfilePicture with source image url string 1`] = `
<div
  className="profile-picture-container profile-picture-container__active"
>
  <div
    className="profile-picture-container--border  profile-picture-container--border__active"
  >
    <img
      alt="Avatar"
      className="profile-picture-container--image profile-picture-container--image__active"
      onClick={[Function]}
      src="daniel_einars-400x500.jpg"
    />
  </div>
</div>
`;
